deno/deepgram-transcribe-video/src/mod.ts

```diff
@@ -0,0 +1,53 @@
+export default async function (req: any, res: any) {
+  const APIkey = req.variables["DEEPGRAM_API_KEY"];
+  const fileUrl = JSON.parse(req.payload)["fileUrl"];
```

Comment:
Make sure to catch errors. Passing an invalid JSON string for payload results in an unhelpful error:

```shell
curl http://localhost:3000/ -d '{"variables": {},"payload": "{\"filUrl\":\"https://static.deepgram.com/examples/interview_speech-analytics.wav\""}' -H "X-Internal-Challenge: secret-key" -H "Content-Type: application/json"
{
  "stdout": "",
  "stderr": "Code file not found."
}
```